tests/gql_behave/tests/memgraph_V1/features/hops_limit.feature

```diff
@@ -0,0 +1,255 @@
+# Copyright 2024 Memgraph Ltd.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+# License, and you may not use this file except in compliance with the Business Source License.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+Feature: Hops Limit
+
+    Scenario: Test hops limit DFS test01
```

Comment:
As I understand here we print partial results. Can we make it clear in Scenario name please?